docs/tutorials/performance.md

```diff
@@ -160,13 +159,24 @@ You can avoid using slirp4netns in the following ways:
   The [socket activation tutorial](https://github.com/containers/podman/blob/main/docs/tutorials/socket_activation.md)
   provides more information about socket activation support in Podman.
 
-* Use the network driver [_pasta_](https://passt.top/passt/about/#pasta). Pasta is under development and currently needs a patched Podman to run.
-
-* Set up the network manually as root. Create a bridge and virtual ethernet pair (VETH). See the [example](https://lists.podman.io/archives/list/podman@lists.podman.io/thread/W6MCYO6RY5YFRTSUDAOEZA7SC2EFXRZE/) posted on the Podman mailing list. See also the section _DIY networking_ in [Podman-Rootless-Networking.pdf](https://podman.io/community/meeting/notes/2021-10-05/Podman-Rootless-Networking.pdf).
+* Set up the network manually as root. Create a bridge and virtual ethernet pair (VETH). For more information, see
+  + An [example](https://lists.podman.io/archives/list/podman@lists.podman.io/thread/W6MCYO6RY5YFRTSUDAOEZA7SC2EFXRZE/) posted on the Podman mailing list
+  + The section _DIY networking_ in [Podman-Rootless-Networking.pdf](https://podman.io/community/meeting/notes/2021-10-05/Podman-Rootless-Networking.pdf)
+  + The GitHub project [neverpanic/podman-rootful-network](https://github.com/neverpanic/podman-rootful-network) that contains a Python script for attaching network interfaces to initialized containers
```

Comment:
This is not supported by us in any way.
I am not strictly against linking this but it should be clear that this is just as DIY as the link above and not supported/recommended by the Podman team.

Reply:
I removed the reference from this PR, just for an easy fix.
It could be always be added later. I plan to try out the instructions in
https://github.com/neverpanic/podman-rootful-network
to see how it works.

Reply:
On a second thought... I'm not sure if it's fundamental to mention that here, but it looks like this is a magic way to avoid the need for root without any further consequences.

Well, with UID 0 in the container and a veth pair, one can open AF_PACKET or AF_RAW sockets on the veth interface and send out arbitrary frames, with spoofed MAC / IP addresses, as well as potentially sniffing on the host's traffic. Maybe it would be worth mentioning that this doesn't implement network isolation, unlike pasta and slirp4netns do.

Reply:
> send out arbitrary frames, with spoofed MAC / IP addresses, as well as potentially sniffing on the host's traffic

Oops, I didn't think of that.

> Maybe it would be worth mentioning that this doesn't implement network isolation, unlike pasta and slirp4netns do.

Yes, I agree.
It also sounds like container UID == 0 versus UID != 0 makes a difference. That could also be mentioned.

@sbrivio-rh if you already have an idea how to improve the PR, feel free to modify the PR directly. I could also try to write something (but I lack a bit the knowledge about these details).

Reply:
> It also sounds like container UID == 0 versus UID != 0 makes a difference. That could also be mentioned.
 
Oh, right, I missed the fact that by default the UID 0 account you get in a Podman container doesn't have the `CAP_NET_ADMIN` capability, it seems.

```
$ podman run --rm -ti alpine sh
/ # ip link add dev x type dummy
ip: RTNETLINK answers: Operation not permitted
/ # id
uid=0(root) gid=0(root) groups=0(root),1(bin),2(daemon),3(sys),4(adm),6(disk),10(wheel),11(floppy),20(dialout),26(tape),27(video)
```

but:

```
$ podman run --privileged --rm -ti alpine sh
/ # ip link add dev x type dummy
/ # id
uid=0(root) gid=0(root) groups=0(root),1(bin),2(daemon),3(sys),4(adm),6(disk),10(wheel),11(floppy),20(dialout),26(tape),27(video)
```

while the UID is zero in both cases.

> @sbrivio-rh if you already have an idea how to improve the PR, feel free to modify the PR directly. I could also try to write something (but I lack a bit the knowledge about these details).

Added as a comment, I wasn't sure how to edit it directly. :)

Reply:
Thanks a lot! I added your text.

Reply:
I noticed the formatting/indentation  looks a bit strange. I'll see if I can improve it.

Reply:
I moved your text before the two references. This improved markdown formatting.